pylustrator/ax_rasterisation.py

```diff
@@ -20,27 +20,31 @@
 # along with Pylustrator. If not, see <http://www.gnu.org/licenses/>
 
 import io
+
 import matplotlib.pyplot as plt
+
 try:  # starting from mpl version 3.6.0
     from matplotlib.axes import Axes
-except:
+except Exception:

```

Comment:
Just adding Exception as the exception does not make the except less of a wildcard. I think here the right thing would be ImportError